courses/models.py

```
@@ -0,0 +1,49 @@
+class Course(models.Model):
+    title = models.CharField(max_length=200)
+    slug = models.SlugField(unique=True)
+    description = models.TextField()
+    instructor = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='courses_taught')
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    is_active = models.BooleanField(default=True)
+    price = models.DecimalField(max_digits=10, decimal_places=2, default=0.00)
+    thumbnail = models.ImageField(upload_to='course_thumbnails/', null=True, blank=True)
+
+    def save(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `save`

Add return type annotation: `None`

(ANN201)